Cannot install Node dependencies with npm5 because of `node-pre-gyp` bug

It looks like with npm5, `node-pre-gyp` is causing the independencies to fail to install:

```bash
node-pre-gyp info it worked if it ends with ok
node-pre-gyp verb cli [ '/usr/local/Cellar/node/8.0.0_1/bin/node',
node-pre-gyp verb cli   '/opt/aframe-site/node_modules/fsevents/node_modules/.bin/node-pre-gyp',
node-pre-gyp verb cli   'install',
node-pre-gyp verb cli   '--fallback-to-build' ]
node-pre-gyp info using node-pre-gyp@https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.6.33.tgz
node-pre-gyp info using node@8.0.0 | darwin | x64
node-pre-gyp verb command install []
node-pre-gyp ERR! install error
node-pre-gyp ERR! stack TypeError: Cannot read property 'version' of null
node-pre-gyp ERR! stack     at Object.module.exports.evaluate (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/util/versioning.js:270:32)
node-pre-gyp ERR! stack     at install (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/install.js:168:31)
node-pre-gyp ERR! stack     at Object.self.commands.(anonymous function) [as install] (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/node-pre-gyp.js:50:37)
node-pre-gyp ERR! stack     at run (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp:79:30)
node-pre-gyp ERR! stack     at Object. (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp:131:1)
node-pre-gyp ERR! stack     at Module._compile (module.js:569:30)
node-pre-gyp ERR! stack     at Object.Module._extensions..js (module.js:580:10)
node-pre-gyp ERR! stack     at Module.load (module.js:503:32)
node-pre-gyp ERR! stack     at tryModuleLoad (module.js:466:12)
node-pre-gyp ERR! stack     at Function.Module._load (module.js:458:3)
node-pre-gyp ERR! System Darwin 15.2.0
node-pre-gyp ERR! command "/usr/local/Cellar/node/8.0.0_1/bin/node" "/opt/aframe-site/node_modules/fsevents/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /opt/aframe-site/node_modules/fsevents
node-pre-gyp ERR! node -v v8.0.0
node-pre-gyp ERR! node-pre-gyp -v vhttps://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.6.33.tgz
node-pre-gyp ERR! not ok
Cannot read property 'version' of null
```

<details>
<pre><code>
% npm i                                               /opt/aframe-site (master)
npm WARN prefer global marked@0.3.6 should be installed with -g
npm WARN prefer global jshint@2.9.4 should be installed with -g

> fsevents@1.1.1 install /opt/aframe-site/node_modules/fsevents
> node install

node-pre-gyp info it worked if it ends with ok
node-pre-gyp verb cli [ '/usr/local/Cellar/node/8.0.0_1/bin/node',
node-pre-gyp verb cli   '/opt/aframe-site/node_modules/fsevents/node_modules/.bin/node-pre-gyp',
node-pre-gyp verb cli   'install',
node-pre-gyp verb cli   '--fallback-to-build' ]
node-pre-gyp info using node-pre-gyp@https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.6.33.tgz
node-pre-gyp info using node@8.0.0 | darwin | x64
node-pre-gyp verb command install []
node-pre-gyp ERR! install error 
node-pre-gyp ERR! stack TypeError: Cannot read property 'version' of null
node-pre-gyp ERR! stack     at Object.module.exports.evaluate (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/util/versioning.js:270:32)
node-pre-gyp ERR! stack     at install (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/install.js:168:31)
node-pre-gyp ERR! stack     at Object.self.commands.(anonymous function) [as install] (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/lib/node-pre-gyp.js:50:37)
node-pre-gyp ERR! stack     at run (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp:79:30)
node-pre-gyp ERR! stack     at Object.<anonymous> (/opt/aframe-site/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp:131:1)
node-pre-gyp ERR! stack     at Module._compile (module.js:569:30)
node-pre-gyp ERR! stack     at Object.Module._extensions..js (module.js:580:10)
node-pre-gyp ERR! stack     at Module.load (module.js:503:32)
node-pre-gyp ERR! stack     at tryModuleLoad (module.js:466:12)
node-pre-gyp ERR! stack     at Function.Module._load (module.js:458:3)
node-pre-gyp ERR! System Darwin 15.2.0
node-pre-gyp ERR! command "/usr/local/Cellar/node/8.0.0_1/bin/node" "/opt/aframe-site/node_modules/fsevents/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /opt/aframe-site/node_modules/fsevents
node-pre-gyp ERR! node -v v8.0.0
node-pre-gyp ERR! node-pre-gyp -v vhttps://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.6.33.tgz
node-pre-gyp ERR! not ok 
Cannot read property 'version' of null

> dtrace-provider@0.8.0 install /opt/aframe-site/node_modules/dtrace-provider
> node scripts/install.js


> spawn-sync@1.0.15 postinstall /opt/aframe-site/node_modules/spawn-sync
> node postinstall


> hexo-util@0.6.0 postinstall /opt/aframe-site/node_modules/hexo-util
> npm run build:highlight

npm ERR! Invalid version: "https://registry.npmjs.org/hexo-util/-/hexo-util-0.6.0.tgz"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cvan/.npm/_logs/2017-07-11T19_31_03_851Z-debug.log

> hexo-util@0.5.3 postinstall /opt/aframe-site/node_modules/hexo-deployer-git/node_modules/hexo-util
> npm run build:highlight

npm ERR! Invalid version: "https://registry.npmjs.org/hexo-util/-/hexo-util-0.5.3.tgz"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cvan/.npm/_logs/2017-07-11T19_31_04_306Z-debug.log

> hexo-util@0.5.3 postinstall /opt/aframe-site/node_modules/hexo-renderer-marked/node_modules/hexo-util
> npm run build:highlight

npm ERR! Invalid version: "https://registry.npmjs.org/hexo-util/-/hexo-util-0.5.3.tgz"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cvan/.npm/_logs/2017-07-11T19_31_05_177Z-debug.log
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@https://registry.npmjs.org/fsevents/-/fsevents-1.1.1.tgz (node_modules/fsevents):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.1 install: `node install`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: hexo-util@https://registry.npmjs.org/hexo-util/-/hexo-util-0.6.0.tgz (node_modules/hexo-util):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: hexo-util@0.6.0 postinstall: `npm run build:highlight`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: hexo-util@https://registry.npmjs.org/hexo-util/-/hexo-util-0.5.3.tgz (node_modules/hexo-deployer-git/node_modules/hexo-util):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: hexo-util@0.5.3 postinstall: `npm run build:highlight`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hexo-util@0.5.3 postinstall: `npm run build:highlight`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hexo-util@0.5.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cvan/.npm/_logs/2017-07-11T19_31_06_558Z-debug.log
</code></pre>

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+package-lock=false

